content/guides/tooling/sdks/sdk-versioning.md

```diff
@@ -79,12 +106,24 @@ Follow detailed migrations guides to migrate from from Box Next Generations SDKs
 
 [java-v10]: https://github.com/box/box-java-sdk/tree/sdk-gen
 [ios-v10]: https://github.com/box/box-ios-sdk/tree/sdk-gen
-[node-v10]: https://github.com/box/box-windows-sdk-v2/tree/sdk-gen
+[node-v10]: https://github.com/box/box-node-sdk/tree/sdk-gen
 [python-v10]: https://github.com/box/box-python-sdk/tree/sdk-gen
 [windows-v10]: https://github.com/box/box-windows-sdk-v2/tree/sdk-gen
 
-[java-migration]: https://github.com/box/box-java-sdk/blob/sdk-gen/migration-guides/from-box-java-sdk-gen-v0-to-box-java-sdk-v10.md
-[python-migration]: https://github.com/box/box-python-sdk/blob/sdk-gen/migration-guides/from-box-python-sdk-gen-v1-to-box-python-sdk-v10.md
-[swift-migration]: https://github.com/box/box-ios-sdk/blob/sdk-gen/migration-guides/from-box-swift-sdk-gen-v0-to-box-ios-sdk-v10.md
-[ts-migration]: https://github.com/box/box-node-sdk/blob/sdk-gen/docs/migration-guides/from-box-typescript-sdk-gen-v1-to-box-node-sdk-v10.md
-[dotnet-migration]: https://github.com/box/box-windows-sdk-v2/blob/sdk-gen/migration-guides/from-dotnet-sdk-gen-v1-to-box-windows-sdk-v10.md
\ No newline at end of file
+[java-migration-v10]: https://github.com/box/box-java-sdk/blob/sdk-gen/migration-guides/from-box-java-sdk-gen-v0-to-box-java-sdk-v10.md
+[python-migration-v10]: https://github.com/box/box-python-sdk/blob/sdk-gen/migration-guides/from-box-python-sdk-gen-v1-to-box-python-sdk-v10.md
+[swift-migration-v10]: https://github.com/box/box-ios-sdk/blob/sdk-gen/migration-guides/from-box-swift-sdk-gen-v0-to-box-ios-sdk-v10.md
+[ts-migration-v10]: https://github.com/box/box-node-sdk/blob/sdk-gen/docs/migration-guides/from-box-typescript-sdk-gen-v1-to-box-node-sdk-v10.md
+[dotnet-migration-v10]: https://github.com/box/box-windows-sdk-v2/blob/sdk-gen/migration-guides/from-dotnet-sdk-gen-v1-to-box-windows-sdk-v10.md
```

Comment:
Link to dotnet is incorrect.

It should be

https://github.com/box/box-windows-sdk-v2/blob/sdk-gen/migration-guides/from-dotnet-sdk-gen-v1-to-box-windows-sdk.md

(no v10 at the end)